.circleci/config.yml

```diff
@@ -49,7 +63,7 @@ jobs:
       - run: 
           name: Install Dependencies
           command:  |
-            npm install shelljs@0.8.4
+            npm install shelljs@0.8.5
```

Comment:
We are still using 0.8.4 (see https://github.com/forcedotcom/SalesforceMobileSDK-Package/search?q=shelljs)

Reply:
Does the version need to match?  `0.8.5` has a security fix.

Reply:
We should update to 0.8.5 across the board then.

Reply:
Yeah, I will send a PR for that too.  Any reason not to merge this PR?

Reply:
Nope.